src/Components/Home/Search.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Search.css';
+
+function Search(props) {
+  const { query, onSearch } = props;
+
+  return (
+    <div className="searchContainer">
+      <input
+        type="text"
+        placeholder="search currencies"
+        value={query}
+        onChange={onSearch}
+      />
```

Comment:
[ Optional Changes ]

- [ ] It would be nice if you could make the search currencies placeholder a bit clear so one knows what is being guided in the search box. You brighten it to light white color a little bit 👍🏽
- [ ] Or better still, you could add a search bar and brighten its color to light white so it's visible 👍🏽

Reply:
Thanks